front/pages/events_map.js

```diff
@@ -22,8 +22,16 @@ export default (mapFactory, api) => {
                     infowindow.close();
                 }
 
+                let content = `<a href="${event.url}" target="_blank">${event.name}</a><br>Organisé par`;
+
+                if ('committee_url' in event) {
+                    content += `<a href="${event.committee_url}" target="_blank">${event.committee_name}</a>`;
+                } else {
+                    content += '&nbsp;' + event.organizer;
```

Comment:
I guess it comes from the templating syntax. Let's keep it like that, it's fine.